modules/sandboxed-containers-common-terms.adoc

```diff
@@ -0,0 +1,27 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+[id="sandboxed-containers-common-terms_{context}"]
+= {sandboxed-containers-first} common terms
+
+The following terms are used throughout the documentation.
+
+Sandbox:: A sandbox is an isolated environment where programs can run. In a sandbox, you can run untested or untrusted programs without risking harm to the host machine or the operating system.
++
+In the context of {sandboxed-containers-first}, sandboxing is achieved by running workloads in a different kernel using virtualization, providing enhanced control over the interactions between multiple workloads that run on the same host.
+
+Pod:: A pod is a construct that is inherited from Kubernetes and {product-title}. It represents resources where containers can be deployed. Containers run inside of pods, and pods are used to specify resources that can be shared between multiple containers.
++
+In the context of {sandboxed-containers-first}, a pod is implemented as a virtual machine. Several containers can run in the same pod on the same virtual machine.
+
+{sandboxed-containers-operator}:: An Operator is a software component that automates operations, which are actions that a human operator could do on the system.
++
+The {sandboxed-containers-operator} is tasked with managing the lifecycle of sandboxed containers on a cluster. It deals with operations, such as the installation and removal of sandboxed containers software and status monitoring.
+
+Kata Containers:: Kata Containers is a core upstream project that is used to build {sandboxed-containers-first}. {sandboxed-containers-first} integrate Kata Containers with {product-title}.
+
+KataConfig:: `KataConfig` objects represents configurations of sandboxed containers. They store information about the state of the cluster, such as the nodes on which the software is deployed.
+
+{op-system} extensions:: {op-system-first} extensions are a mechanism to install optional {product-title} software. The {sandboxed-containers-operator} uses this mechanism to deploy sandboxed containers on a cluster.
+
+Runtime class:: A `RuntimeClass` object describes which runtime can be used to run a given workload. A runtime class that is named `kata` is installed and deployed by the {sandboxed-containers-operator}. The runtime class contains information about the runtime that describes resources that the runtime needs to operate, such as the pod overhead.
```

Comment:
Is "the pod overhead" clear enough? I've not run across this usage before and can't tell if it means "overhead" as in "above" or as in "debt".

Reply:
I believe it is in terms of debt. @jensfr could you clarify?

Reply:
The term is a standard Kubernetes term with its own page. You could link to it: https://kubernetes.io/docs/concepts/scheduling-eviction/pod-overhead/